src/assets/style.css

```diff
@@ -1,3 +1,7 @@
+tr.first-bracket:not(:first-child) > td {
+  border-top-width: 5px !important; /* Overrides ".table-bordered > tbody > tr > td" rule. */
```

Comment:
On Chrome 56, the `!important` does not seem necessary when we have `:not(:first-child)`.

Are we sure this is necessary ?